core/templates/dev/head/pages/exploration_editor/exploration_metadata_modal_directive.html

```diff
@@ -6,48 +6,8 @@
   <p>
     Just a few more things before you publish:
```

Comment:
This is unrelated to the fix, but could we actually get rid of this "Just a few more things before you publish:" line and instead put it in the above line instead of "Add Some Details"?